frontend/src/pages/Home/Home.tsx

```diff
@@ -51,17 +43,9 @@ export const Home = () => {
         onChange={e => setFilterValue(e.target.value.toLowerCase())}
         value={filterValue}
       />
+      <Loader isLoading={isLoading} />
       <PokemonList pokemonList={pokemonList} filterValue={filterValue} />
-    </div>
-  )
-}
-
-const PokemonList = ({ pokemonList, filterValue }: PokemonList) => {
-  return (
-    <div className={styles.cardList}>
-      {filterPokemonsByName(pokemonList, filterValue).map(({ name, id, weight, height }) => {
-        return <Pokemon name={name} id={id} key={id} weight={weight} height={height} />
-      })}
+      <Error error={error} />
```

Comment:
- <!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
    
    **🎯 suggestion (non-blocking):** <subject>I think fetching the pokemon list (and handling loading and error) should be move in the PokemonList component, directly related to this use case. Moreover, a page component should only be responsible for listing the components to display, and should not have complex behavior.
    
    <discussion>